src/template.html

```diff
@@ -139,39 +139,38 @@
           />
         </div>
         <ul
-          class="flex overflow-hidden gap-4 font-semibold text-2xl rounded-lg px-8 py-6 w-fit text-slate-100 bg-black/30 backdrop-blur-md sm:hidden"
+          class="flex items-center overflow-hidden sm:gap-1 gap-4 font-semibold sm:text-xl text-2xl rounded-2xl sm:mx-4 px-8 py-2 sm:px-2  w-fit text-slate-100 bg-black/30 backdrop-blur-md"
         >
-          <li class="">
+          <li>
             <a
-              class="p-6 hover:bg-white hover:text-gamify-black hover:rounded-xl duration-300"
-              href="#"
+              class="nav-link flex active p-6 pr-4 hover:bg-white hover:text-gamify-black hover:rounded-xl duration-300"
+              href="#content"
               >Games
               <span id="game-counter"></span>
             </a>
           </li>
-          <li class="">
+          <li>
             <a
-              class="p-6 hover:bg-white hover:text-gamify-black hover:rounded-xl duration-300"
-              href="#"
+              class="nav-link flex p-6 hover:bg-white hover:text-gamify-black hover:rounded-xl duration-300"
+              href="#content"
               >Trailers</a
             >
           </li>
           <li class="">
             <a
-              class="p-6 hover:bg-white hover:text-gamify-black hover:rounded-xl duration-300"
-              href="#"
+              class="nav-link p-6 hover:bg-white hover:text-gamify-black hover:rounded-xl duration-300"
+              href="#content"
               >Contact</a
             >
           </li>
         </ul>

```

Comment:
### Cool NavBar Feel

This is a great improvement to user experience and I love that it is sticky so the user does not have to click a dropdown to see the menu on mobile plus, the whole design gives a cool feel.